phpstan.dist.neon

```diff
@@ -0,0 +1,16 @@
+parameters:
+    level: 1
+    paths:
+        - src
+    excludePaths:
+        - src/Behat/Testwork/EventDispatcher/TestworkEventDispatcherSymfonyLegacy.php
+        - src/Behat/Testwork/EventDispatcher/TestworkEventDispatcherSymfony5.php
+    ignoreErrors:
+        -
+            identifier: class.notFound
```

Comment:
We ignore this error in this class as it refers to some PHPUnit classes that may not be available depending on the PHPUnit version